core-java-modules/core-java-arrays-guides/src/main/java/com/baeldung/printarrays/PrintArrayJava.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.printarrays;
+import java.util.Arrays;
+
+public class PrintArrayJava {
+
+    // Print array content using a for loop
+    public String printArrayUsingForLoop(String[] empArray) {
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < empArray.length; i++) {
+            result.append(empArray[i]).append(" ");
+        }
+        return result.toString().trim();
+    }
+
+    // Print array content using a for-each loop
+    public String printArrayUsingForEachLoop(String[] empArray) {
+        StringBuilder result = new StringBuilder();
+        for (String arr : empArray) {
+            result.append(arr).append("\n");
+        }
+        return result.toString().trim();
+    }
+
+    // Print array content using Arrays.toString
+    public String printArrayUsingToString(int[] empIDs) {
+        return Arrays.toString(empIDs);
+    }
+
+    // Print array content using Arrays.asList
+    public String printArrayUsingAsList(String[] empArray) {
+        return Arrays.asList(empArray).toString();
+    }
+
+    // Print array content using Streams
+    public String printArrayUsingStreams(String[] empArray) {
+        StringBuilder result = new StringBuilder();
+        Arrays.stream(empArray).forEach(e -> result.append(e).append("\n"));
+        return result.toString().trim();
+    }
+
+    // Print 2D array content using nested loops
+    public String print2DArrayUsingNestedLoops(int[][] exampleArr) {
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < exampleArr.length; i++) {
+            for (int j = 0; j < exampleArr[i].length; j++) {
+                result.append(exampleArr[i][j]).append(" ");
+            }
+            result.append("\n");
```

Comment:
Same here, enhanced for loops.
```suggestion
        for (int[] ints : exampleArr) {
            for (int anInt : ints) {
                result.append(anInt)
                    .append(" ");
            }
            result.append("\n");
        }
```

Reply:
As you mentioned, there's no need to discuss 2D arrays since a dedicated guide has already been published on this topic, so I removed that code from my article.